src/domains/transaction/pages/SendVote/FormStep.tsx

```diff
@@ -63,49 +63,50 @@ export const FormStep = ({
 	}, [setValue, fees]);
 
 	return (
-		<section data-testid="SendVote__form-step">
+		<section data-testid="SendVote__form-step" className="space-y-8">
```

Comment:
check if your merge re-added properties that were explicitly removed, as I don't think this class is still needed

Reply:
https://github.com/ArkEcosystem/desktop-wallet/pull/3278/files#diff-c2977ee1a07d719090ea7c0ac1631987b9c3ccf0211c96f2032000e4a95eef7a for reference